src/makeCalculator.js

```diff
@@ -37,7 +37,31 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    add(num1, num2) {
+      return num1 + num2;
+    },
```

Comment:
![image](https://user-images.githubusercontent.com/59209337/228750256-543f2e86-2cdc-4cad-bcb4-e0f59b5958c3.png)

Reply:
I implemented this in `operate`  function. If I set value in the `add` function, the callback has no sense.

Reply:
sorry, it's my fault, your solution is OK